docs/source/references.bib

```diff
@@ -328,6 +340,17 @@ @InProceedings {makinen-2019-exact
   month =	 Sep
 }
 
+@Article {makinen-2020-collaborative,
```

Comment:
Is this cited somewhere?

Reply:
That is a good point. I added the two references mentioned in the bm4d pypi webpage but used only one in the docstrings. I had the intention of using two like bm3d but ran into some issues.

I saw the bm3d doc-string cites two papers, one is cited normally and another saying the software was released with this paper. I am not sure how to determine if a code is released with a particular paper.

Reply:
If both references are equally relevant to BM4D, then cite them both in the docs. Otherwise probably worth removing the one that isn't cited for consistency since all our references are cited at least once.